examples/lora/sensor-lorawan/CayenneLPP.cc

```diff
@@ -0,0 +1,219 @@
+/*
+ * The Cayenne Low Power Payload (LPP) provides a convenient and
+ * easy way to send data over LPWAN networks such as LoRaWAN. The Cayenne LPP
+ * is compliant with the payload size restriction, which can be lowered down to
+ * 11 bytes, and allows the device to send multiple sensor data at one time.
+ *
+ * CayenneLPP is also support by TTN:
+ *     https://www.thethingsindustries.com/docs/integrations/payload-formatters/cayenne/
+ *
+ * This code is direct from: https://github.com/myDevicesIoT/CayenneLPP
+ *
+ * GPL-3.0 license
+ * See https://github.com/myDevicesIoT/CayenneLPP for license details
```

Comment:
I am so not a lawyer, but I don't think we have any other GPL in this repo, so I'm a bit wary (mostly out of ignorance) of whether this can cause any complications for us / others.

Right this second not passing judgement one way or another, just elevating attention to this so we can talk/think about it.

Reply:
At least two things should be changed:

- The GPL license should be included in full somewhere in the repo, like in a `LICENSE.GPL` file.
- An `SPDX-License-Identifier` tag near the top of the file so this file doesn't accidentally get redistributed in a non-GPL-compliant manner